config/config.yaml

```diff
@@ -1,7 +1,6 @@
-Hypervisor_Provider : ""
-iso_url : ""
-minishift_PATH : ""
-minishift_lib_MODULE : ""
+Hypervisor_Provider : "kvm"
+minishift_PATH : "/home/agajdosi/minishift/minishift"
+minishift_lib_MODULE : "/home/agajdosi/git/cdk-minishift-testsuite/library/minishift_testlib.py"
```

Comment:
Agree, fixed in https://github.com/agajdosi/cdk-minishift-testsuite/commit/a65f445d540ea81d75f1013a14af36db871f6731